python/pip_install/private/test/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
+
+diff_test(
+    name = "srcs_diff_test",
+    failure_message = "Please run `bazel run @rules_rust//python/pip_install/private:srcs_module.install`",
```

Comment:
s/rules_rust/rules_python/

Reply:
In addition I think we should assume anyone re-running this rule will be a rule developer working inside rules_python so it should be `bazel run //python/pip_install/private:srcs_module.<verb>`